src/lib/core/datetime.c

```diff
@@ -163,9 +163,12 @@ tm_to_datetime(struct tnt_tm *tm, struct datetime *date)
 size_t
 datetime_strptime(struct datetime *date, const char *buf, const char *fmt)
 {
-	assert(date != NULL);
```

Comment:
Agreed with @ligurio that we still prefer to have some assertions enabled here in (average) Debug mode, but to have those assertions disabled if fuzzing mode is activated. At the moment it looks like we want to introduce some fuzzing-friendly `*_ASSERT` macro.

Reply:
OTOH, we do need to protect `datetime_strptime` against `NULL`s coming from outer world, and, even, with some sort of assertions left here, we still do need to have checks here for the `Release`/`RelWithDebInfo` compile modes.

Reply:
Updated, these asserts were reverted.